pokemongo_bot/cell_workers/recycle_items.py

```diff
@@ -67,47 +76,122 @@ def should_run(self):
         :return: True if the recycling process should be run; otherwise, False.
         :rtype: bool
         """
-        if inventory.Items.get_space_left() < (DEFAULT_MIN_EMPTY_SPACE if self.min_empty_space is None else self.min_empty_space):
+        if inventory.items().get_space_left() < (DEFAULT_MIN_EMPTY_SPACE if self.min_empty_space is None else self.min_empty_space):
```

Comment:
get_space_left() is a class method, thus it should be called by Items an not items(), please revert